<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindow</class>
 <widget class="QMainWindow" name="mainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1374</width>
    <height>563</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Guarantor Search</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #fbfbfb;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableView" name="guarantorTable">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>1351</width>
      <height>471</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #f1f1f1;
border: 1px solid transparent;
border-radius: 10px;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="searchButton">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>20</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #0077c2;
    color: white;
    border: 1px solid transparent;
    border-radius: 10px;
    padding: 5px 10px;
}

QPushButton:hover {
    border: 1px solid white;
}

QPushButton:pressed {
    background-color: #005f9e;
    padding-top: 6px;
    padding-bottom: 4px;
}

QPushButton:disabled {
    background-color: lightgray;
    color: gray;
    border: 1px solid gray;
}
</string>
    </property>
    <property name="text">
     <string>Search</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="loanNumber">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>20</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
    border: none;
    border-bottom: 1px solid black;
    background-color: transparent;
    color: black;
}

QLineEdit:disabled {
    border: none;
    border-bottom: 1px solid lightgray;
    background-color: lightgray;
    color: lightgray;
}
</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_22">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(241, 241, 241);
border: 1px solid #ef9a9a;
border-radius: 10px;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>Loan Number</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>loanNumber</tabstop>
  <tabstop>searchButton</tabstop>
  <tabstop>guarantorTable</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
